lib/mobile_app_backend/predictions/pub_sub.ex

```diff
@@ -4,20 +4,30 @@ defmodule MobileAppBackend.Predictions.PubSub.Behaviour do
 
   @type predictions_for_stop :: %{Stop.id() => JsonApi.Object.full_map()}
 
-  @type subscribe_response :: %{
+  @type subscribe_stop_response :: %{
           predictions_by_stop: %{Stop.id() => %{Prediction.id() => Prediction.t()}},
           trips: %{Trip.id() => Trip.t()},
           vehicles: %{Vehicle.id() => Vehicle.t()}
         }
 
+  @type subscribe_trip_response :: %{
```

Comment:
suggestion(non-blocking): add the new `trip_id` field here

Reply:
Sorry, I don't understand what you mean?

Reply:
There is a trip_id field added to the map [here](https://github.com/mbta/mobile_app_backend/pull/235/files#diff-03648ca268be035bc968db4cbf1e34012cfd2fba57e08254a1bbc6f47a444957R235), I think it would be worth adding to the typespec

Reply:
Ohh, okay yes, updated